plugins/repository-hdfs/build.gradle

```diff
@@ -134,6 +134,10 @@ for (String fixtureName : ['hdfsFixture', 'haHdfsFixture', 'secureHdfsFixture',
 
     args miniHDFSArgs.toArray()
   }
+
+  // TODO: The task configuration block has side effects that require it currently to be always executed.
+  // Otherwise tests start failing. Therefore we enforce the task creation for now.
+  tsk.get()
```

Comment:
Should we just use `create()` then? This seems even more confusing.

Reply:
keeping `create()` might look like an oversight. I think this way makes it more explicit that there is something wrong with the task setup.

Reply:
Cool, let's create an issue to look into what the task configuration block is doing that's required for test setup.

Reply:
done here: https://github.com/elastic/elasticsearch/issues/56948